commands/maggy.md

```
@@ -0,0 +1,94 @@
+    open "${URL}" 2>/dev/null || true
+    exit 0
+  fi
+  sleep 1
+done
+echo "Maggy didn't come up in 15s. Check ~/.maggy/maggy.log"
+```
+
+### 5. Report status
+
+Show:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)